### PR TITLE
Migrate to machine-api-provider-gcp

### PR DESCRIPTION
**Pull request by alexander-demicev**
In order to migrate from cluster-api-provider-gcp to machine-api-provider-gcp I added similar test suite to new repo.

### CODE REVIEW

File: ci-operator/config/openshift/machine-api-provider-gcp/OWNERS [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+# See the OWNERS docs: https://git.k8s.io/community/contributors/guide/owners.md
+
+approvers:
+- alexander-demichev
+- elmiko
+- enxebre
+- fedosin
+- joelspeed
+- lobziik
+- michaelgugino
```
Comment (elmiko) [inline ci-operator/config/openshift/machine-api-provider-gcp/OWNERS:11]: maybe we should take Mike G off these owners?

File: ci-operator/jobs/openshift/machine-api-provider-gcp/OWNERS [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+# See the OWNERS docs: https://git.k8s.io/community/contributors/guide/owners.md
+
+approvers:
+- alexander-demichev
+- elmiko
+- enxebre
+- fedosin
+- joelspeed
+- lobziik
+- michaelgugino
```
Comment (elmiko) [inline ci-operator/jobs/openshift/machine-api-provider-gcp/OWNERS:11]: same here

File: core-services/prow/02_config/openshift/machine-api-provider-gcp/OWNERS [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+# See the OWNERS docs: https://git.k8s.io/community/contributors/guide/owners.md
+
+approvers:
+- alexander-demichev
+- elmiko
+- enxebre
+- fedosin
+- joelspeed
+- lobziik
+- michaelgugino
```
Comment (elmiko) [inline core-services/prow/02_config/openshift/machine-api-provider-gcp/OWNERS:11]: same

**Review by elmiko — COMMENTED** [review]
i think we should clean up the owners files, but otherwise it looks fine to me

**Comment by JoelSpeed** [comment]
/lgtm

Looking at the PR history, I'm content the jobs are stable enough

**Pull request merged**